Rakefile

```diff
@@ -27,4 +30,19 @@ task :internal_investigation do
   abort('RuboCop failed!') unless result.zero?
 end
 
-task default: [:spec, :internal_investigation]
+desc 'Build config/default.yml'
+task :build_config do
+  sh('bin/build_config')
+end
+
+desc 'Confirm config matches is unchanged'
+task confirm_config: :build_config do
+  _, stdout, _, process =
+    Open3.popen3('git diff --exit-code config/default.yml')
+
+  if ENV.key?('CI') && !process.value.success?
```

Comment:
Hm well I just didn't want it to fail the rake task unless we were on CI. It still dumps the new config locally so you would see it while staging your code